### ISSUE TITLE
Don't display new loader over non-image placeholder?

### ISSUE THREAD

**Issue by choumx** [ampproject/amphtml#24625]
https://github.com/ampproject/amphtml/issues/20237#issuecomment-479038688

Should be a simple fix.

`b/141275200`

**Comment by choumx** [comment]
On second thought, maybe it's better to defer to the page author. This avoids deviation from legacy behavior, and authors can easily add `noloading` attribute.

/cc @ampproject/wg-ui-and-a11y

**Comment by nainar** [comment]
Currently we show the loaders regardless of the placeholder. 
This has been the case even before the loaders switch. 

If people want to switch to not having a loader they can add the `noloading` attr like you mentioned Will. 

However, if we change to not having the loader when a placeholder is shown there is no opt in if a user wants to change that behaviour.
